### PR TITLE
websocat: init at 1.3.0

### PR DESCRIPTION
**Pull request by thoughtpolice**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/websocat/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}:
+
+rustPlatform.buildRustPackage rec {
+  name = "websocat-${version}";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner  = "vi";
+    repo   = "websocat";
+    rev    = "refs/tags/v${version}";
```
Comment (srhb) [inline pkgs/tools/misc/websocat/default.nix]:
```suggestion
    rev    = "v${version}";
```

... alternatively an explanatory comment for why the release tarball doesn't work? :)

**Review by srhb — APPROVED** [review]

File: pkgs/tools/misc/websocat/default.nix (hunk 1 above)
Comment (thoughtpolice) [reply pkgs/tools/misc/websocat/default.nix]: No, I simply always use `refs/tags` -- I was under the impression there's no difference, i.e. that fetchFromGitHub always acquires an archive tarball of some tree snapshot, though I might obviously wrong about that.

**Comment by thoughtpolice** [comment]
Closed in 37fe6bf7868d7c84ae567de446a46f3fd42a8f63.

File: pkgs/tools/misc/websocat/default.nix (hunk 1 above)
Comment (srhb) [reply pkgs/tools/misc/websocat/default.nix]: No, you're right and I'm wrong. Thanks. :)

**Pull request closed without merging**